frontend/src/metabase/static-viz/components/XYChart/utils/values.ts

```diff
@@ -0,0 +1,33 @@
+import { getY } from "./series";
+
+import type { TextProps } from "@visx/text";
+import type { HydratedSeries } from "../types";
+import { measureText } from "metabase/static-viz/lib/text";
+
+// Use the same logic as in https://github.com/metabase/metabase/blob/72d23d2b8e5a1f93b288e5a8738758c9b05d3766/frontend/src/metabase/visualizations/lib/chart_values.js#L318
+export function getValueStep(
```

Comment:
Almost the same logic in https://github.com/metabase/metabase/blob/72d23d2b8e5a1f93b288e5a8738758c9b05d3766/frontend/src/metabase/visualizations/lib/chart_values.js#L318

One caveat is that this is a simplified version of that logic and it doesn't render different series differently unlike the referenced logic where each series could have a different step since different series could have a different amount of data points.

This will be addressed in a followed-up PR.